### DISCUSSION TITLE
Some nice Stools - bunch of tools

### DISCUSSION THREAD

**Discussion by ELPatitoLoco** [TAServers/issues#35] [Addon Suggestions]
This are a bunch of tools that are mostly funny and and just lightweight.
[Face Pose me - Allows you to change your face like a ragdoll](https://steamcommunity.com/sharedfiles/filedetails/?id=104542300)
[Resizer  - Resize props with a tool](https://steamcommunity.com/sharedfiles/filedetails/?id=116892991)
[Smart Painter - Paints Smartly :thumbsup: ](https://steamcommunity.com/sharedfiles/filedetails/?id=126566296)
[Swep Kit - helps you in making a swep](https://steamcommunity.com/sharedfiles/filedetails/?id=109724869)

**Comment by Derpius** [comment]
Prop resizer is unbelievably unstable

**Comment by Derpius** [comment]
Face poser is outdated and appears partially/completely non-functional from the comments and discussions

**Reply by ELPatitoLoco** [reply]
works on zoltins tho

**Comment by Derpius** [comment]
Smart painter is _also_ ancient and appears to possibly not work (shift clicking). I imagine the code is awful and given it needs to traverse and compare props joined by constraints or in an area (i haven't actually looked yet, but the average gmod dev cannot comprehend algorithms and data structures)

**Comment by Derpius** [comment]
SWEP one appears to be a tool for people to create sweps with? Or am I missing something

**Reply by ELPatitoLoco** [reply]
yeah, its in e2bi aswell

**Reply by Derpius** [reply]
I mean like, to generate swep code, not to make a weapon on the fly ingame
